File: WebApp/Descriptor9BackPortal/src/types/Product.ts
export interface Product {
  id: string;
  title: string;
  descriptionTranslations: Langs;
  category: string;
  imageUrl: string;
}

export interface Langs {
  english: string;
  spanish: string;
  dutch: string;
  ukrainian: string;
  romanian: string;
  serbian: string;
}

File: Services/DescriptorGeneratorAPI/DescriptorGeneratorAPI/Models/Product.cs
namespace DescriptorGeneratorAPI.Models
{
    public class Product
    {
        public int Id { get; set; }
        public string? Title { get; set; }
        public DescriptionTranslations? DescriptionTranslations { get; set; }
        public string? Category { get; set; }
        public string? ImageUrl { get; set; }
    }
}
